app_data/vocabularies/experiments.yaml

```diff
@@ -1,3726 +1,7328 @@
-- id: ACCMOR NA32
-  title:
-    en: ACCMOR NA32
-- id: ACE AD-4
+- id: IS749
+  title:
+    en: IS749
+  description: Emission Mossbauer spectroscopy of topological kagome magnets
+  link: https://isolde.cern/active-experiments
+- id: nTOF79
+  title:
+    en: nTOF79
+  description: Measurement of 28,29,30 Si(n,y) capture cross-sections to explain isotopic
+    abundances in presolar grains
+  link: None
```

Comment:
Kept values that were also `None` in the harvested json. Not sure maybe we can just skip them